src/background.js

```diff
@@ -3,7 +3,7 @@ import 'src/omnibar'
 
 
 function openOverview() {
-    browser.tabs.create({
+    browser.tabs.update({
```

Comment:
This will make the overview open up in the currently active tab when the shortcut combination is entered, right? There doesn't seem to be any discussion about this yet, so what do you guys (@urbancvek @oliversauter) think should happen from a user POV? 

Personally, as a user, I'd rather press the shortcut and have a new tab open up to not disturb my current tab. Or maybe open in the same tab, but have the option of holding Shift with the existing shortcut to make it open in a new tab. What do you think?